opentelemetry-exporter-cloud-trace/tests/test_cloud_trace_exporter_integration.py

```diff
@@ -0,0 +1,82 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import socket
+import subprocess
+import unittest
+
+import grpc
+from google.cloud.trace_v2 import TraceServiceClient
+from google.cloud.trace_v2.gapic.transports import trace_service_grpc_transport
+from opentelemetry.exporter.cloud_trace import CloudTraceSpanExporter
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace.export import Span, SpanExportResult
+from opentelemetry.trace import SpanContext, SpanKind
+
+
+class TestCloudTraceSpanExporter(unittest.TestCase):
+    def setUp(self):
+        # Find a free port to spin up our server at.
+        sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        sock.bind(("localhost", 0))
+        self.port = sock.getsockname()[1]
+        sock.close()
+
+        # Start our mock server.
+        args = "./mock_server-x64-linux -address=localhost:" + str(self.port)
```

Comment:
I think it might be a little cleaner to invoke with a non-relative path and assume the binary exists in `$PATH` (see also my below comment in tox.ini).

That way this code could be reused by users who want to write their own tests against the mock server for their projects.

Reply:
Done